### PR TITLE
fix: always reset time when setting range

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

Fix ​an issue where when setting a range, the time would be reset to the start time of the range regardless of the set time

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/timepicker/index.js [hunk 1]
```diff
@@ -768,7 +768,9 @@ var TimePicker = defineClass(
      * @private
      */
     applyRange: function(beginHour, beginMin, endHour) {
-      this.setTime(beginHour, beginMin);
+      if (this.isLaterThanSettedTime(beginHour, beginMin)) {
```
Comment (lja1018) [inline src/js/timepicker/index.js]: `setted`가 없는 단어라 `isLaterThanSetTime`이면 될 것 같아요.

**Review by lja1018 — COMMENTED** [review]
리뷰완료합니다.

File: src/js/timepicker/index.js [hunk 2]
```diff
@@ -836,6 +838,17 @@ var TimePicker = defineClass(
       minute <= END_NUMBER_OF_MINUTE;
     },
 
+    /**
+     * Compare given time with setted time
```
Comment (jwlee1108) [inline src/js/timepicker/index.js]: 여기도 수정해주면 좋겠네요 `setted` -> `set`

**Review by jwlee1108 — COMMENTED** [review]
리뷰 완료합니다.

**Review by adhrinae — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**